app/src/pages/EducadoAdmin.tsx

```diff
@@ -1,89 +1,88 @@
 import { useState } from "react";
 import useSWR from "swr";
 import { useNavigate, Link, useLocation } from 'react-router-dom';
-
+import {BACKEND_URL} from "../helpers/environment"
 // Services
 import AuthServices from "../services/auth.services";
 
-// Interfaces
-import { CCApp } from '../interfaces/CCApp'
-
 // Components
 import Loading from "./Loading";
 import Layout from "../components/Layout";
 import { PageDescriptor } from "../components/PageDescriptor";
 
-import {BACKEND_URL} from "../helpers/environment"
-
 
 
 const EducadoAdmin = () => {
+    //Variable to detect and determine the search term used to filter applications
     const [searchTerm, setSearchTerm] = useState('')
-
-    // Used to navigate to the detailed view for the single applicant
+    
+    //Location and navigation 
     let navigate = useNavigate();
     let location = useLocation();
 
+    //Get data from the relevant route
     const { data, error } = useSWR(
-        `${BACKEND_URL}/api/applications?approved=false`,
+        `${BACKEND_URL}/api/applications`,
```

Comment:
Why is data set to some URL to the backend? We shouldn't use the backend url in this file at all, since it all should be handled in the service files

Reply:
This has now been fixed in both `EducadoAdmin.tsx` and `SingleApplicantView.tsx`